src/ethereumProofPointIssuer.ts

```diff
@@ -0,0 +1,275 @@
+import { ProofPointIssueResult } from "./proofPointIssueResult";
+import { ethers } from "ethers";
+import { EthereumAddress, ProofPointId } from "./ethereumProofPointEvent";
+import { ProofPoint } from "./proofPoint";
+import {
+  IpfsStorageProvider,
+  IpfsStorageProviderSettings,
+  StorageProvider,
+} from "./storage";
+import canonicalizeJson from "canonicalize";
+import localISOdt = require("local-iso-dt");
+import { EthereumAddressResolver } from "./ethereumAddressResolver";
+import {
+  EthereumProofPointRegistry,
+  ETHEREUM_PROOF_TYPE,
+} from "./ethereumProofPointRegistry";
+import { EthereumProofPointRegistryRoot } from "./ethereumProofPointRegistryRoot";
+import { RealHttpClient } from "./httpClient";
```

Comment:
For clarity, could external imports come first, then list local imports?

Reply:
Done